tests/api_tests/collections/test_views.py

```diff
@@ -0,0 +1,622 @@
+# -*- coding: utf-8 -*-
+from nose.tools import *  # flake8: noqa
+
+from framework.auth.core import Auth
+from website.models import Node
+from website.util.sanitize import strip_html
+from api.base.settings.defaults import API_BASE
+from tests.base import ApiTestCase, fake
+from tests.factories import UserFactory, FolderFactory, DashboardFactory, NodeFactory, ProjectFactory
+
+
+class TestCollectionsList(ApiTestCase):
+    def setUp(self):
+        ApiTestCase.setUp(self)
+        self.user = UserFactory.build()
+        self.user.set_password('justapoorboy')
+        self.user.save()
+        self.basic_auth = (self.user.username, 'justapoorboy')
+
+        self.deleted = FolderFactory(is_deleted=True, creator=self.user)
+        self.collection = FolderFactory(creator=self.user, is_public=True, )
+        self.url = "/{}collections/".format(API_BASE)
+
+    def test_only_returns_non_deleted_collections(self):
+        res = self.app.get(self.url, auth=self.basic_auth)
+        node_json = res.json['data']
+
+        ids = [node['id'] for node in node_json]
+        assert_not_in(self.deleted._id, ids)
+        assert_in(self.collection._id, ids)
+
+    def test_return_collection_list_logged_in_user(self):
+        res = self.app.get(self.url, auth=self.basic_auth)
+        assert_equal(res.status_code, 200)
+        ids = [node['id'] for node in res.json['data']]
+        assert_in(self.collection._id, ids)
+
+    def test_return_collection_list_logged_out_user(self):
+        res = self.app.get(self.url, auth=self.basic_auth)
```

Comment:
This is not testing a logged out user. It passes because the assert is incorrect.